builds/delete-serverless-service-build-pack/Jenkinsfile

```diff
@@ -850,6 +850,8 @@ def resetCredentials(credsId) {
 	echo "resetting AWS credentials"
 	sh "aws configure set profile.${credsId}.aws_access_key_id XXXXXXXXXXXXXXXXXXXXXXXXXX"
 	sh "aws configure set profile.${credsId}.aws_secret_access_key XXXXXXXXXXXXXXXXXXXXXX"
```

Comment:
if we are deleting the creds, then replacing the creds with XXXXX seems redundant

Reply:
yeah, will remove

Reply:
Done